<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNetwork</class>
 <widget class="QWidget" name="FormNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, QCheckBox, QSpinBox{
	color: rgb(255, 255, 255);
}
QWidget:disabled{
	color: rgb(120, 120,120);
}
QPushButton:disabled{
	border-image: url(:/stylesheet/button_disabled.png);
}
/*PushButton*/
QPushButton{
	color: rgb(180, 188, 142);
	border-image: url(:/stylesheet/button.png);
}
QPushButton:hover, QPushButton:focus{	
	color: rgb(255, 255, 255);	
	border-image: url(:/stylesheet/button_hover.png);
}
QPushButton:pressed{
	padding-top: 2px;
	border-image: url(:/stylesheet/button_pressed.png);
}
QPushButton:disabled{
	color: rgb(120, 120, 120);
	border-image: url(:/stylesheet/button_disabled.png);
}
/*ComboBox, QSpinBox and LineEdit*/
QLineEdit, QComboBox, QSpinBox{	
	padding: 0 8px;
	color: white;
	border: 1px solid white;
	border-radius: 2px;
	background-color: rgba(255, 255, 255, 5);
	selection-background-color: rgba(180, 190, 140, 40);
}
QComboBox::down-arrow{
	border-image: url(:/stylesheet/down_arrow.png);
	width: 15px;
	height: 15px;
 }
QSpinBox::down-arrow{
	bottom:2px;
	right:2px;
	border-image: url(:/stylesheet/down_arrow.png);
	width: 15px;
	height: 15px;
 }
QSpinBox::up-arrow{
	top: 2px;
	right:2px;
	border-image: url(:/stylesheet/up_arrow.png);
	width: 15px;
	height: 15px;
 }
QComboBox::down-arrow:on{ 
     top: 1px;
     right: 1px;
 }
QComboBox::drop-down {
    width: 30px;
	border-left-width: 1px;    
 }
QSpinBox::up-button,QSpinBox::down-button{
	background-color: rgba(255, 255, 255, 0);
 }
 QAbstractItemView {	
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
 }
/*ChecBox*/
QCheckBox {
    spacing: 2px;
}
QCheckBox::indicator {
    width: 30px;
    height: 30px;
}
QCheckBox::indicator:unchecked {
    image: url(:/stylesheet/checkbox_unchecked.png);
}
QCheckBox::indicator:unchecked:hover {
    image: url(:/stylesheet/checkbox_unchecked_hover.png);
}
QCheckBox::indicator:unchecked:pressed {
    image: url(:/stylesheet/checkbox_unchecked_pressed.png);
}
QCheckBox::indicator:unchecked:disabled {
    image: url(:/stylesheet/checkbox_unchecked_disabled.png);
}
QCheckBox::indicator:checked {
    image: url(:/stylesheet/checkbox_checked.png);
}
QCheckBox::indicator:checked:hover {
    image: url(:/stylesheet/checkbox_checked_hover.png);
}
QCheckBox::indicator:checked:pressed {
    image: url(:/stylesheet/checkbox_checked_pressed.png);
}
QCheckBox::indicator:checked:disabled {
    image: url(:/stylesheet/checkbox_unchecked_disabled.png);
}
#_w_title{	
	background-color: rgba(0, 0, 0, 80);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="_w_title" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <spacer name="horizontalSpacer_19">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/resources.qrc">:/stylesheet/network.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_20">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/menu/menu_top.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Constantia</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Servidor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Puerto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Seguridad</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="_le_server">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="inputMask">
               <string>000.000.000.000</string>
              </property>
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="_sb_port">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>9090</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="_cb_use_ssl">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Revue BT</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Usar SSL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_pb_accept">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Revue BT</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Aceptar</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_pb_cancel">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Revue BT</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
            <property name="shortcut">
             <string>Esc</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/resources.qrc">:/menu/menu_bottom.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
